### PR TITLE
Update index.html (issue #7)

### PR DESCRIPTION
**Pull request by Archies11**
I have added the correct soundtrack for the songs "Bang a drum" and "Magic"
The working link : https://archies11.github.io/Selena-Gomez-Discography/

### CODE REVIEW

**Comment by OrionStar25** [comment]
Please post the working link as well. Read the `readme` file for instructions.

**Comment by Archies11** [comment]
Mam Do I post the link here in these comments?

**Comment by OrionStar25** [comment]
Yes, post it here :)

File: index.html [hunk 1]
```diff
@@ -278,15 +278,15 @@ <h2><i>Kiss And Tell</i></h2>
 			</audio>
 
 
-			<a href="#" class="list-group-item">Bang A Drum</a>
+			<a href="https://www.youtube.com/watch?v=3sGXoDxoTPs" class="list-group-item">Bang A Drum</a>
```
Comment (OrionStar25) [inline index.html]: The link doesn't need to change. You have to change the default song in the music player. Right now only one type of song is assigned to each player. Download the song and add it to the player.

File: index.html [hunk 2]
```diff
@@ -278,15 +278,15 @@ <h2><i>Kiss And Tell</i></h2>
...
              <audio controls>
  			<source src="songs/Kygo Selena Gomez - It Aint Me (with Selena Gomez) (Audio).mp3"
          	type='audio/mp3'>
   <!-- The next line will only be executed if the browser doesn't support the <audio> tag-->
  			<p>Your user agent does not support the HTML5 Audio element.</p>
 			</audio>
 
-            <a href="#" class="list-group-item">Magic</a>
+            <a href="https://www.youtube.com/watch?v=Qtb11P1FWnc" class="list-group-item">Magic</a>
```
Comment (OrionStar25) [inline index.html]: Same here.

**Review by OrionStar25 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Archies11) [reply index.html]: Mam i have made the necessary changes.
link: https://archies11.github.io/Selena-Gomez-Discography/

Comment (OrionStar25) [reply index.html]: Alright, so you've made the changes the Fetish and Bad Liar. Works good :dancer:

**Review by OrionStar25 — APPROVED** [review]

**Pull request merged**